tests/sles4sap/saptune.pm

```diff
@@ -33,6 +34,10 @@ sub run {
     # saptune is not installed by default on SLES4SAP 12 on ppc64le
     zypper_call "-n in saptune" if (get_var('OFW') and is_sle('<15'));
 
+    my $saptune_version = script_output "rpm -q --qf '%{VERSION}' saptune";
+    # Skip test if saptune version is < 2
+    return if (version->parse($saptune_version) < version->parse(2.0));
```

Comment:
No need to skip the whole test. If this is failing because saptune's list of solutions and/or list of notes is different, you could move L39 after L55 and you keep a basic check